mobilecoind/api/proto/mobilecoind_api.proto

```diff
@@ -32,7 +32,6 @@ service MobilecoindAPI {
     rpc GetRequestCode (GetRequestCodeRequest) returns (GetRequestCodeResponse) {}
     rpc ReadTransferCode (ReadTransferCodeRequest) returns (ReadTransferCodeResponse) {}
     rpc GetTransferCode (GetTransferCodeRequest) returns (GetTransferCodeResponse) {}
-    rpc GetAddressRequestCode (GetAddressRequestCodeRequest) returns (GetAddressRequestCodeResponse) {}
```

Comment:
should we then also drop the message definitions?
```
// Encode a URL into a base-58 "MobileCoin Address Request Code".
message GetAddressRequestCodeRequest {
    string url = 1;
}

message GetAddressRequestCodeResponse {
    string b58_code = 1;
}
```

Does this proposed change impact our ability to use a single URI to pass "intents" into apps? How will a scanned URI be flagged as an address request?

Reply:
Yes, I've dropped them, I think we should cross this bridge when we have a better idea of exactly how that flow will work.